extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -456,6 +456,7 @@ class AmpFetcher {
     return this.xhr_
       .fetchJson(url, {
         credentials: 'include',
+        prerenderSafe: true,
```

Comment:
Let's also flip the `isPrerenderSafe()` method to true?

Reply:
Updated!